pouch/convertor/cri.go

```diff
@@ -0,0 +1,222 @@
+package convertor
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"regexp"
+	"strconv"
+	"strings"
+
+	localtypes "github.com/pouchcontainer/d2p-migrator/pouch/types"
+	"github.com/pouchcontainer/d2p-migrator/utils"
+
+	runtime "github.com/alibaba/pouch/cri/apis/v1alpha2"
+)
+
+const (
+	// PouchContainerTypeLabelKey label to identify whether a container is a sandbox
+	// or container
+	PouchContainerTypeLabelKey = "io.kubernetes.pouch.type"
+	// PouchContainerTypeLabelSandbox specify this container is a sandbox
+	PouchContainerTypeLabelSandbox = "sandbox"
+	// PouchContainerTypeLabelContainer specify this container is a container
+	PouchContainerTypeLabelContainer = "container"
+
```

Comment:
We will add a label in the future: containerLogPathLabelKey = "io.kubernetes.container.logpath"